docs/containers-signature.5.md

```diff
@@ -1,10 +1,10 @@
-% atomic-signature(5) Atomic signature format
+% container-signature(5) Container signature format
 % Miloslav Trmač
 % March 2017
 
-# Atomic signature format
+# Container signature format
 
-This document describes the format of “atomic” container signatures,
+This document describes the format of “container” signatures,
```

Comment:
I don’t think we need the quotes after dropping the distinguishing qualifier.

Reply:
Fixed